src/components/OurCourses/useStyles.js

```diff
@@ -0,0 +1,51 @@
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles(({ typography, breakpoints }) => ({
+  root: {
+    backgroundColor: "#DFDFDF",
```

Comment:
Already defined in [theme](https://github.com/CodeForAfrica/PesaYetu/blob/08f271db28c6417e188bfb0c0ebd7375684465e7/src/theme/index.js#L44)

Reply:
Fixed